### PR TITLE
[js_hw3_loops][Task_01,task_02,task_03] Accomlished all tasks"

### PR DESCRIPTION
**Pull request by Maximus2806**

### CODE REVIEW

File: js_hw_3/task_01.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+};
+
+// 2. Написать скрипт, который выведет 5 строк в консоль таким образом, чтобы в первой строчке выводилось :), во второй :):) и так далее
+//   Пример в консоли:
+//   :)
+//   :):)
+//   :):):)
+//   :):):):)
+//   :):):):):)
+let smile = ":)";
```
Comment (anatoly-karpovich) [inline js_hw_3/task_01.js:14]: Use const

File: js_hw_3/task_01.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+for (let number = 0; number < 5; number++) {
```
Comment (anatoly-karpovich) [inline js_hw_3/task_01.js]: better use "i" as iterator

File: js_hw_3/task_01.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    console.log(smile);
+    smile = smile + ":)"
+};
+
+//   Для Оксаны и ее друзей: также реализовать циклом while
+//   Рекоммендация: попробуйте метод .repeat()
+let smile2 = ":)"
+let number2 = 1
```
Comment (anatoly-karpovich) [inline js_hw_3/task_01.js]:
```suggestion
const smile2 = ":)"
const number2 = 1
```

File: js_hw_3/task_02.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    if (!+value || typeof(value) === "string"){
+        return "Invalid input"
+    } else {
+        let i = 0
+    while (value >= 1024){
+        value = value / 1024
+        i ++
+    }
+    switch(i) {
+        case 0: return value + " bytes";
```
Comment (anatoly-karpovich) [inline js_hw_3/task_02.js]: better use return after switch block.

File: js_hw_3/task_03.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
+// 1*. Создать программу, которая будет принимать на вход СЛОВО (создать переменную со словом), 
+//   и выводить в консоль количество гласных и согласных букв в этом слове. 
+//   Ответ должен выводиться шаблонным литералом вида word contains x vowels and y consonants
+
+function countLetters(str){
+    if(str !== undefined && str !== null && str.length > 0){
+        let word = str.toLowerCase();
+        let vovels = "aeiou";
+        let consonants = "bcdfghjklmnpqrstvwxz";
```
Comment (anatoly-karpovich) [inline js_hw_3/task_03.js]:
```suggestion
        const word = str.toLowerCase();
        const vovels = "aeiou";
        const consonants = "bcdfghjklmnpqrstvwxz";
```

File: js_hw_3/task_03.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+// 2**. Написать программу, которая видоизменяет принимаемое слово (переменная со словом) 
+//   шифром ЦЕЗАРЯ (посмотреть в википедии) со сдвигом на 1 в любую из сторон. 
+//   Направление шифрования задается переменной offset, которая может быть +1 и -1.
+//   Например let str = 'AbC'; let offset = -1, result = 'ZaB';
+//   Например let str = 'ZzZ'; let offset = 1, result = 'AaA';
+
+function encryptCezar(str, offset){
+    if(str!== undefined && str!== null && str.length > 0 && (offset === -1 || offset === 1)){
+        let alphabet = 'abcdefghijklmnopqrstuvwxyz';
+        let alphabetUp = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```
Comment (anatoly-karpovich) [inline js_hw_3/task_03.js]:
```suggestion
        const alphabet = 'abcdefghijklmnopqrstuvwxyz';
        const alphabetUp = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

**Review by anatoly-karpovich — APPROVED** [review]
Well done!

**Review by Sapphireexe — APPROVED** [review]

**Pull request merged**